src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/proxyintegrations/mekanism/DigitalMinerIntegration.java

```diff
@@ -26,75 +31,118 @@ protected String getName() {
     }
 
     @LuaFunction
-    public int getDelay() {
-        return getTileEntity().getDelay();
+    public final int getToMine() {
+        return getTileEntity().cachedToMine;
     }
 
     @LuaFunction
-    public int getRadius() {
-        return getTileEntity().getRadius();
+    public final boolean isRunning() {
+        return getTileEntity().running;
     }
 
     @LuaFunction
-    public void setRadius(int newRadius) {
-        getTileEntity().setRadiusFromPacket(newRadius);
+    public final boolean getAutoEject() {
+        return getTileEntity().inverse;
     }
 
     @LuaFunction
-    public int getMinY() {
-        return getTileEntity().getMinY();
+    public final void setAutoEject(boolean eject) {
+        if(!(eject == getTileEntity().doEject))
+            getTileEntity().toggleAutoEject();
     }
 
     @LuaFunction
-    public void setMinY(int newMinY) {
-        getTileEntity().setMinYFromPacket(newMinY);
+    public final boolean getAutoPull() {
+        return getTileEntity().doPull;
     }
 
     @LuaFunction
-    public int getMaxY() {
-        return getTileEntity().getMaxY();
+    public final void setAutoPull(boolean pull) {
+        if(!(pull == getTileEntity().doPull))
+            getTileEntity().toggleAutoPull();
+    }
+
+    @LuaFunction
+    public final boolean getSilkTouch() {
+        return getTileEntity().getSilkTouch();
     }
 
     @LuaFunction
-    public void setMaxY(int newMaxY) {
-        getTileEntity().setMaxYFromPacket(newMaxY);
+    public final void setSilkTouch(boolean silk) {
+        if(!(silk == getTileEntity().getSilkTouch()))
+            getTileEntity().toggleSilkTouch();
     }
 
     @LuaFunction
-    public void toggleSilkTouch() {
-        getTileEntity().toggleSilkTouch();
+    public final void start(){
+        TileEntityDigitalMiner tile = ((TileEntityDigitalMiner) getTileEntity());
+        // just calling tile.start() causes the miner searcher thread to don´t find any blocks to mine
+        // doing it like that ensures that it´s called as the client would do it (and works)
+        Mekanism.packetHandler.sendToServer(new PacketGuiInteract(GuiInteraction.START_BUTTON, tile));
     }
 
     @LuaFunction
-    public void toggleInverse() {
-        getTileEntity().toggleInverse();
+    public final void stop(){
+        TileEntityDigitalMiner tile = ((TileEntityDigitalMiner) getTileEntity());
+        Mekanism.packetHandler.sendToServer(new PacketGuiInteract(GuiInteraction.STOP_BUTTON, tile));
+    }
+    
+    @LuaFunction
+    public final void reset(){
+        TileEntityDigitalMiner tile = ((TileEntityDigitalMiner) getTileEntity());
```

Comment:
You don't need to create an extra var for the digital miner, you can use `getTileEntity` for `sendToServer`